lxd/storage/drivers/utils_cgo.go

```diff
@@ -307,6 +306,7 @@ func PrepareLoopDev(source string, flags int) (*os.File, error) {
 	if loopFd >= 0 {
 		return os.NewFile(uintptr(loopFd), C.GoString((*C.char)(cLoopDev))), nil
 	}
+
```

Comment:
I assume this is a separate commit on purpose requested by another maintainer and is not something that you introduced. If this is just a whitespace fix that already existed make it the first commit of the series. Otherwise don't introduce the whitespace error in the first place. :)